cpp/ql/src/Best Practices/RuleOfThree.qhelp

```diff
@@ -1,24 +0,0 @@
-<!DOCTYPE qhelp PUBLIC
-  "-//Semmle//qhelp//EN"
-  "qhelp.dtd">
-<qhelp>
-
-
-<overview>
-<p>This query finds classes that define a destructor, a copy constructor, or a copy assignment operator, but not all three of them. The compiler generates default implementations for these functions, and since they deal with similar concerns it is likely that if the default implementation of one of them is not satisfactory, then neither are those of the others.</p>
-
-<p>The query flags any such class with a warning, and also display the list of generated warnings in the result view.</p>
-
-</overview>
-<section title="How to Address the Query Results">
-<p>Explicitly define the missing functions.</p>
-
-
-
-
-
-</section>
-<references>
-<li><a href="http://en.wikipedia.org/wiki/Rule_of_three_(C%2B%2B_programming)">Wikipedia article</a>
-
-</li></references></qhelp>
```

Comment:
This and some of the other `.qhelp`'s are a bit sparse, but not completely lacking useful information.  I can't help but wonder if the long term solution should be to add the things they're missing, such as code samples and more detailed references - unless we decide to deprecate the query.

Reply:
I agree that this one is also somewhat borderline. In that case, I'll revert this deletion and update the section titles to meet current expectations. After that, will it be okay to publish it as-is or should it be added to the list for exclusion?

Most of the others don't go beyond what's already included in the query description.